UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterTile.cs

```diff
@@ -297,9 +297,9 @@ public virtual void OnDespawnServer(DespawnInfo info)
 	#endregion
 
 
-	public void ServerSetLocalPosition(Vector3Int value)
+	public void ServerSetLocalPosition(Vector3Int value, bool OverRideCheck = false)
```

Comment:
remember that all variables that are a parameter of a function will be in ``camelCase``